app/controller/sdl/RCModulesController.js

```diff
@@ -44,6 +44,12 @@ SDL.RCModulesController = Em.Object.create({
       LIGHT: 'lightModels',
       HMI_SETTINGS: 'hmiSettingsModels'
     },
+    
+    /**
+     * @description List of available UUIDs for SEAT module.
+     * @type {List}
+     */
+    // seatUUIDs: [],
```

Comment:
@YarikMamykin you can remove this at all

Reply:
@AKalinich-Luxoft removed in https://github.com/LuxoftSDL/sdl_hmi/pull/13/commits/8e050fe8f937ae904e6ce7ea0bae7863d2e9bdd7